tutor/templates/apps/openedx/settings/partials/common_all.py

```diff
@@ -150,9 +150,12 @@
 # These warnings are visible in simple commands and init tasks
 import warnings
 
-from django.utils.deprecation import RemovedInDjango50Warning, RemovedInDjango51Warning
-warnings.filterwarnings("ignore", category=RemovedInDjango50Warning)
-warnings.filterwarnings("ignore", category=RemovedInDjango51Warning)
+try:
+    from django.utils.deprecation import RemovedInDjango50Warning, RemovedInDjango51Warning
```

Comment:
Could you please just add a comment that includes the "REMOVE-AFTER-V18" string? and explain why this try/except was still needed in v17.